packages/adminrouter/extra/dcos-adminrouter-slave.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Admin Router Agent: A high performance web server and a reverse proxy server
```

Comment:
I followed the example of the mesos units, which use `slave` in filenames and unit names but `agent` in descriptions. https://github.com/dcos/dcos/blob/master/packages/mesos/extra/dcos-mesos-slave.service#L2